spec/fixtures/users.yml

```diff
@@ -16,99 +16,70 @@ affiliate_admin:
 
 affiliate_added_by_another_affiliate:
   <<: *DEFAULTS
-  email: affiliate_added_by_another_affiliate@fixtures.org
-  is_affiliate_admin: false
-  is_affiliate: true
   first_name: Invited Affiliate Manager
   last_name: Smith
-  approval_status: approved
```

Comment:
I removed settings that were redundant to either the fixture defaults, or the db/model defaults:
```yaml
is_affiliate_admin: false
is_affiliate: true
approval_status: approved
```